data.tf

```
@@ -0,0 +1,7 @@
+data "aws_ami" "rempup_ami_version" {
+  owners = ["amazon"]
+  filter {
+    name   = "image-id"
+    values = ["ami-09c5ba4f838d8684a"]
```

Consider using something like:
```suggestion
    values = ["ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server-*"]
```

which is nicer to read and easy to bump up the OS version, but also will get the latest AMI when needed.
But if you use this you'll need to ignore AMI changes when you create the instance e.g.:
```
lifecycle {
    ignore_changes = [
      image_id, tags
    ]
  }
```